examples/features/records.effekt.md

```diff
@@ -0,0 +1,32 @@
+# Tuples & records
+
+A tuple is written as comma-separated list delimited by parentheses:
+
+```
+val t = ("a", 42)
+```
+
+For destructing a tuple, you may use pattern matching:
+
+```effekt:repl
+t match { case (first, second) => first }
+```
```

Comment:
Nitpicks:
- should we also show the type of a tuple? (perhaps something like "t has type `(String, Int)" as a comment?
- you can also use the _other_ deconstruction syntax:
```
val (first, second) = t
```